This patch is created by Oracle to:

  1. Remove reference in named(8) to lwresd(8) which is not, and has
     never been, provide with Solaris distribution.

  2. Add reference to dns-server(8s), SMF manual page.

diff -r 752254461f9c bin/named/named.8
--- a/bin/named/named.8 Mon Apr 18 11:28:30 2016 +0100
+++ b/bin/named/named.8 Thu Apr 21 10:15:34 2016 +0100
@@ -358,7 +358,7 @@
 \fBnamed-checkconf\fR(8),
 \fBnamed-checkzone\fR(8),
 \fBrndc\fR(8),
-\fBlwresd\fR(8),
+\fBdns-server\fR(8s),
 \fBnamed.conf\fR(5),
 BIND 9 Administrator Reference Manual\&.
 .SH "AUTHOR"
diff -r 752254461f9c doc/arm/man.named.html
--- a/doc/arm/man.named.html    Mon Apr 18 11:28:30 2016 +0100
+++ b/doc/arm/man.named.html    Thu Apr 21 10:59:57 2016 +0100
@@ -344,7 +344,7 @@
       <span class="citerefentry"><span class="refentrytitle">named-checkconf</span>(8)</span>,
       <span class="citerefentry"><span class="refentrytitle">named-checkzone</span>(8)</span>,
       <span class="citerefentry"><span class="refentrytitle">rndc</span>(8)</span>,
-      <span class="citerefentry"><span class="refentrytitle">lwresd</span>(8)</span>,
+      <span class="citerefentry"><span class="refentrytitle">dns-server</span>(8s)</span>,
       <span class="citerefentry"><span class="refentrytitle">named.conf</span>(5)</span>,
       <em class="citetitle">BIND 9 Administrator Reference Manual</em>.
     </p>
